SharePointFramework/ProjectExtensions/src/projectSetup/index.ts

```diff
@@ -276,13 +276,19 @@ export default class ProjectSetup extends BaseApplicationCustomizer<IProjectSetu
         data: {},
         level: LogLevel.Info
       })
+
+      const templateFileName = (await sp.web.select("Title", "AllProperties").expand("AllProperties").get())['AllProperties']['pptemplate'];
```

Comment:
I'd prefer `pp_template` for consistency, but this is not major. It will be easier to read with `pp_` prefix though, imo.

Reply:
https://github.com/Puzzlepart/prosjektportalen365/pull/461/commits/bc67f72731e01b25e80c26d439d5ea361a2f9c9d